src/Algebra/Morphism/Structures.agda

```diff
@@ -68,7 +68,6 @@ module MagmaMorphisms (M₁ : RawMagma a ℓ₁) (M₂ : RawMagma b ℓ₂) wher
       ; surjective     = surjective
       }
 
-
```

Comment:
The `style-guide` indicates that there *should* be two blank lines after each internal `module`, although that rule is not observed rigidly throughout even this module, and in any case, arguably, the big section headings function more than adequately as separators... so not sure what to do here?

Reply:
I think that big section headings are indeed more than adequate as separators - and we should update the style guide accordingly.